plugins/inputs/x509_cert/x509_cert.go

```diff
@@ -204,25 +218,47 @@ func getTags(cert *x509.Certificate, location string) map[string]string {
 	return tags
 }
 
+// copied from refreshFilePaths() in plugins/inputs/file/file.go
+func (c *X509Cert) expandFilePathsToUrls() error {
+	var urls []*url.URL
+
+	for _, globpath := range c.globpaths {
+		files := globpath.Match()
+		if len(files) <= 0 {
+			return fmt.Errorf("could not find file: %v", globpath)
+		}
```

Comment:
Isn't this a bit harsh? Do we always expect a matching file for the glob paths? In not we could just `continue` here and maybe issue a debug-log-entry.

Reply:
you are right totally not suitable